main.js

```diff
@@ -17,3 +17,101 @@ navLinks.forEach((ele) => ele.addEventListener('click', () => {
 }));
 
 
+
+/*popup*/
+
+const Project = [
+  {
+    id: 0,
+    img: './imgages/Snapshoot-Portfolio-4.png',
+    header: 'Multi-Post Stories',
+    lists: ['CANOPY', 'Back End', '2015'],
+    paragraph: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.It has survived not only five centuries,',
+    plang: ['HTML', 'CSS', 'JavaScript'],
+  },
+  {
+    id: 1,
+    img: './imgages/Snapshoot-Portfolio.png',
+    header: 'Facebook 360',
+    lists: ['FACEBOOK', 'Back End', '2015'],
+    paragraph: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.It has survived not only five centuries,',
+    plang: ['HTML', 'CSS', 'JavaScript', 'Ruby on rails'],
+  },
+  {
+    id: 2,
+    img: './imgages/Snapshoot-Portfolio-1.png',
+    header: 'Uber Navigations',
+    lists: ['UBER', 'Back End', '2015'],
+    paragraph: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.It has survived not only five centuries,',
+    plang: ['HTML', 'CSS', 'JavaScript', 'Ruby on rails'],
+  },
+  {
+    id: 3,
+    img: './imgages/Snapshoot-Portfolio-2.png',
+    header: 'Multi-Post Stories',
```

Comment:
while you did a good job working on the project requirements you did not stick to the sigma design you picked. Please try as much to stick to the figma design you picked. 

This is your popup for both mobile and Desktop view:
![Screenshot 2023-02-10 at 3 20 26 PM](https://user-images.githubusercontent.com/79506105/218104086-936a6499-bbbe-418d-b422-bf882cb5d3df.png)
![Screenshot 2023-02-10 at 3 20 41 PM](https://user-images.githubusercontent.com/79506105/218104285-ea8beb46-8a3c-429b-a9f0-27f1158faa6c.png)

Please stick to this design:

for mobile:
![Screenshot 2023-02-10 at 3 24 30 PM](https://user-images.githubusercontent.com/79506105/218104429-23dd0ce9-f388-4368-8fe4-b32d72e5fee2.png)


for desktop:

![Screenshot 2023-02-10 at 3 24 14 PM](https://user-images.githubusercontent.com/79506105/218104541-fa8ee591-0fd2-4a39-96a9-f0082f7ddf1d.png)


- Kindly Add images as well to your popup window please stick to the figma template as much as possible.

- The see source and see project button should be functional please add links ignorer to be redirected to th e source of the project and also to be redirected to the project itself